### PR TITLE
Add keyboard shortcuts to the Config/Keybinding chapter.

### PR DESCRIPTION
**Pull request by maxim-uvarov**
I believe these shortcuts should be in the same section as shell shortcuts, so I added them there.

### CODE REVIEW

**Review by ellie — CHANGES_REQUESTED** [review]
Just one to adjust, but otherwise all good! Thanks

File: docs/docs/config/key-binding.md [hunk 1]
```diff
@@ -63,3 +65,28 @@ atuin init fish | source
 bind \cr _atuin_search
 bind -M insert \cr _atuin_search
 ```
+
+## Atuin UI shortcuts
+
+| Shortcut                                  | Action                                                                                                             |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| ctrl + r                                  | Cycle through filter modes                                                                                         |
+| ctrl + s                                  | Switch the search mode, changes the engine and possibly other interface settings                                   |
```
[the remark this answers is not in the document]
Comment (maxim-uvarov) [reply docs/docs/config/key-binding.md]: done! thanks:)

File: docs/docs/config/key-binding.md [hunk 2]
```diff
@@ -63,3 +65,28 @@ atuin init fish | source
...
+| ctrl + u                                  | Clear the current line                                                                                             |
+| ctrl + n / ctrl + j                       | Select the next item on the list                                                                                   |
+| ctrl + p / ctrl + k / up arrow            | Select the previous item on the list                                                                               |
+| page down                                 | Scroll search results one page down                                                                                |
+| page up                                   | Scroll search results one page up                                                                                  |
+| enter                                     | Select highlighted item                                                                                            |
+| ↓                                         | Return original or return query depending on settings if no entries are found, or select the next item on the list |
```
Comment (conradludgate) [inline docs/docs/config/key-binding.md:92]: Maybe we can duplicate the line here to be a bit simpler
```suggestion
| ↓ (with no entry selected)                | Return original or return query depending on settings                                                              |
| ↓                                         | Select the next item on the list                                                              |
```
(this last line can be merged with the `ctrl + n / ctrl + j` line)

File: docs/docs/config/key-binding.md [hunk 3]
```diff
@@ -63,3 +65,28 @@ atuin init fish | source
...
+| ctrl + e / end                            | Move the cursor to the end of the line                                                                             |
+| ctrl + backspace / ctrl + alt + backspace | Remove the previous word / remove the word just before the cursor                                                  |
+| ctrl + delete / ctrl + alt + delete       | Remove the next word or the word just after the cursor                                                             |
+| ctrl + w                                  | Remove the word before the cursor even if it spans across the word boundaries                                      |
+| ctrl + u                                  | Clear the current line                                                                                             |
+| ctrl + n / ctrl + j                       | Select the next item on the list                                                                                   |
+| ctrl + p / ctrl + k / up arrow            | Select the previous item on the list                                                                               |
```
Comment (conradludgate) [inline docs/docs/config/key-binding.md:88]: `up arrow` is inconsistently named in relation to the other arrow key lines

**Review by conradludgate — COMMENTED** [review]
Looks quite nice, thanks!

File: docs/docs/config/key-binding.md (hunk 3 above)
Comment (maxim-uvarov) [reply docs/docs/config/key-binding.md]: Thank you! 
I tried to replace UTF symbols with emojis. But on my mac and in the web Right arrow looks diffrent from others. :(

**Review by ellie — APPROVED** [review]
Thank you!

Seeing as this is your first time contributing, if you would like a holographic contributors-only Atuin sticker, then please fill out this [form!](https://notionforms.io/forms/contributors-stickers)

Either way - thank you so much for the time and effort here ✨

**Pull request merged**